src/Robo/Commands/Tests/BehatCommand.php

```diff
@@ -126,6 +126,12 @@ public function behatDefinitions($options = ['mode' => 'l']) {
 
   /**
    * Launches PHP's internal web server via `drush run-server`.
+   *
+   * @command tests:run-server
+   * @description This initiates a webserver that we can use for behat or other testing.
```

Comment:
You can leave this out, the command will inherit `Launches PHP's internal web server via drush run-server`